packages/vsce/src/extension.ts

```diff
@@ -177,6 +178,22 @@ export async function activate(context: ExtensionContext) {
   });
 
   context.subscriptions.concat(getCommands(treeDataProv, treeview));
+
+  return {
+    api: {
+      configuration: {
+        supportedResources: [
+          CicsCmciConstants.CICS_CMCI_LOCAL_FILE,
+          CicsCmciConstants.CICS_PROGRAM_RESOURCE,
+          CicsCmciConstants.CICS_CMCI_LOCAL_TRANSACTION,
+          CicsCmciConstants.CICS_TCPIPSERVICE_RESOURCE,
+          CicsCmciConstants.CICS_LIBRARY_RESOURCE,
+          CicsCmciConstants.CICS_URIMAP,
+          CicsCmciConstants.CICS_CMCI_TASK,
```

Comment:
Is there a reason we've only got a subset of the reasources in the tree, or should we add the rest? (7 out of 9)

<img width="163" alt="Screenshot 2025-05-16 at 12 37 51 PM" src="https://github.com/user-attachments/assets/a27fa978-cb9f-4447-98c3-48fb59350f12" />